cdi/embedded/src/test/java/org/infinispan/cdi/embedded/test/distexec/DistributedExecutorCDITest.java

```diff
@@ -62,9 +62,6 @@ public void testInvocationUsingImpliedInputCacheRunnable() throws Exception {
 
    static class SimpleCallable implements Callable<Integer>, Serializable, ExternalPojo {
 
-      /** The serialVersionUID */
-      private static final long serialVersionUID = -8589149500259272402L;
```

Comment:
True, the question is kind of pointless, because the OP is worried about compatibility of his serialized class across different JDK versions/implementations, and that's what the JLS is for.

I definitely wouldn't think twice about integrating a PR that added a `Serializable` class without a `serialVersionUID`. But since it was already here, I don't think there's anything to gain from removing it, either.